packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -33,6 +39,10 @@ export const EventMeta = () => {
   if (hideEventTypeDetails) {
     return null;
   }
+  // If we didn't pick a time slot yet, we load bookingData via SSR so bookingData should be set
+  // Otherwise we load seatedEventData from useBookerStore
+  const bookingSeatAttendeesQty = seatedEventData?.attendees || bookingData?.attendees.length;
+  const eventTotalSeats = seatedEventData?.seatsPerTimeSlot || event?.seatsPerTimeSlot;
```

Comment:
Important change, depends if you get here via time slot picking or via SSR direct page.